server/src/routes/api/v1/userActivityRouter.js

```diff
@@ -0,0 +1,28 @@
+import express from "express"
+
+import { User } from "../../../models/index.js"
+
+const userActivityRouter = new express.Router({ mergeParams: true })
+
+userActivityRouter.get("/", async (req, res) => {
+    const userData = req.user
+
+    try {
+        const user = await User.query().findById(userData.id)
+        user.salads = await user.$relatedQuery("salads")
+        user.reviews = await user.$relatedQuery("reviews")
```

Comment:
doesn't look like the reviews are displayed on the users profile page? we should only send data that is being used